controller/index.js

```diff
@@ -0,0 +1,132 @@
+const service = require("../service/index.js");
+
+const genereteJSON = (status, code, dataKey, valueKey) => ({
+  status,
+  code,
+  data: { [dataKey]: valueKey },
+});
+
+const get = async (req, res, next) => {
+  try {
+    const contacts = await service.getContacts();
+    res.json(genereteJSON("success", 200, "contacts", contacts));
+  } catch (error) {
+    console.log(error);
+    next(error);
+  }
+};
+
+const getById = async (req, res, next) => {
+  const { contactId } = req.params;
+
+  try {
+    const contact = await service.getContactById(contactId);
+    res.json(genereteJSON("success", 200, "contact", contact));
+  } catch (error) {
+    console.log(error);
+    res
```

Comment:
Albo `next` albo `res`. Nie mogą być dwa. Bo na samym końcu next sprawi że w server.js tam gdzie masz 4 arguemnty middleware zostanie zwrócona odpowiedź do frontendu.

Reply:
Zrozumiałem, tutaj dodatkowe pytanie, jeśli nie używam `next` to pomijam podanie parametru next w funkcji? A także gdy nie używam `req` to  parametry funckji wyglądają (_, res, next). Analogicznie gdy nie używam, np. `next`.

Reply:
Jeśli nie używasz next to tak `można` go pominąć, podobnie z innymi kluczami.